src/reducers/versions.tsx

```diff
@@ -1256,21 +1276,27 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
         throw new Error(`Version missing for versionId: ${versionId}`);
       }
 
-      const { expandedPaths } = version;
+      const { expandedPaths } = state;
+
+      if (!expandedPaths) {
+        throw new Error('Expanded paths are undefined');
```

Comment:
https://github.com/mozilla/addons-code-manager/blob/d83d73b383bfdaac4c9f239f22ed67266fe27535/src/reducers/versions.tsx#L1275

We should not get this error in real, since we should get "missing version" error before that. To test this line, I have not way to prepare the state with a version loaded and expandedPaths undefined. Or should I manually set the expandedPaths `undefined`?

Reply:
Ah, right, that's tricky. It's ok to ignore it for now (TS covers it well). You can add this so that it's not distracting in the coverage report:

```diff
diff --git a/src/reducers/versions.tsx b/src/reducers/versions.tsx
index 616f66d..58f22bf 100644
--- a/src/reducers/versions.tsx
+++ b/src/reducers/versions.tsx
@@ -1279,6 +1279,10 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
       const { expandedPaths } = state;
 
       if (!expandedPaths) {
+        // TODO: this is not possible to test until expanded paths are
+        // decoupled from versions.
+        // See https://github.com/mozilla/addons-code-manager/issues/1218
+        /* istanbul ignore next */
         throw new Error('Expanded paths are undefined');
       }
 
```